composer.json

```
@@ -25,7 +25,7 @@
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
     "fzaninotto/faker": "^1.8",
     "lucatume/wp-browser": "^3.1.4",
-    "lucatume/wp-snapshot-assertions": "^1.1.0",
+    "lucatume/wp-snaphot-assertions": "^1.1.0",
```

```suggestion
    "lucatume/wp-snapshot-assertions": "^1.1.0",
```

https://github.com/lucatume/wp-snapshot-assertions this one, right?

---

[The name is historically incorrect](https://github.com/lucatume/wp-snapshot-assertions/blob/master/composer.json#L2); the correct Composer name to use `snapHot-assertions`.